evil-collection.el

```diff
@@ -166,6 +166,24 @@ this confusing. It will be included if
   :type '(repeat (choice symbol sexp))
   :group 'evil-collection)
 
+(defcustom evil-collection-key-whitelist '()
+  "List of keys that may be used by evil-collection.
+This is a list of strings that are suitable for input to
+`kbd'. If there are no keys in the list, the whitelist will be ignored."
```

Comment:
GNU style is to put double-spacing between sentences.

Reply:
ok, but two spaces are not being used consistently in the current package.

Reply:
We should fix this.